api_tests/types/smack-my-jasmine-up/index.d.ts

```diff
@@ -0,0 +1,7 @@
+declare module "smack-my-jasmine-up" {
```

Comment:
Is there an alternative to this lib that provide types?

Reply:
Not that I know of.